content/docs/general/supportedBrowsers.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I believe you need to add a reference to this in `content/docs/_index.md` and `content/docs/general/_index.md`. It's also not showing up in the left-side nav at the preview since it's marked as `draft: true`: https://deploy-preview-43--examtools-docs.netlify.app/docs/general/

Reply:
I intend that to stay draft; I didn't actually intend it to be in this PR, but worked on top of the same branch accidentally, and then decided crowdsourcing an answer to the table problem wouldn't necessarily be bad.

Reply:
git rm the file and make a commit and push on this branch so we aren't checking in stuff that's not part of this PR.